### ISSUE TITLE
Add support for rotated logs with do not use `copytrucate` semantics

### ISSUE THREAD

**Issue by bobtiernay-okta** [bgreenlee/pygtail#36]
Currently only `copytruncate` like semantics are supported. However, other systems such as logback (Java) use `renamecreate` semantics.

**Comment by bobtiernay-okta** [comment]
To fix this, the following patch can be applied to `next()`:

Change:

```python
if self._rotated_logfile:
```
to
```python
if self._rotated_logfile or (self._filehandle().tell() == fstat(self._filehandle().fileno()).st_size and fstat(self._filehandle().fileno()).st_ino != stat(self.filename).st_ino):
```

**Comment by bobtiernay-okta** [comment]
May be a duplicate of https://github.com/bgreenlee/pygtail/issues/34.

**Comment by brifordwylie** [comment]
@bobtiernay-okta +1 on this. Would be nice to support 'renamecreate' use case.

**Comment by arekm** [comment]
These seem to be related:
https://github.com/bgreenlee/pygtail/pull/12
https://github.com/bgreenlee/pygtail/pull/21

Default logrotate tool used on Linux distros (https://github.com/logrotate/logrotate) doesn't use copytruncate semantis, so supporting renamecreate looks important.

**Comment by bgreenlee** [comment]
Fixed in #37
